core/src/consensus/tendermint/engine.rs

```diff
@@ -332,7 +332,7 @@ impl ConsensusEngine for Tendermint {
     }
 }
 
-fn is_term_changed(header: &Header, parent: &Header, term_seconds: u64) -> bool {
+pub(crate) fn is_term_changed(header: &Header, parent: &Header, term_seconds: u64) -> bool {
```

Comment:
How about `pub(super)`?
The core crate seems too public.

Reply:
Or moving it to the parent module(the tendermint module) looks good to me.

Reply:
`pub(super)` seems to be okay for now. I'll fix it.